.github/workflows/ci.yml

```diff
@@ -136,3 +141,19 @@ jobs:
           PHP_CS_FIXER_IGNORE_ENV: ${{ matrix.PHP_CS_FIXER_IGNORE_ENV }}
           PHP_CS_FIXER_FUTURE_MODE: 1
         run: php php-cs-fixer --diff --dry-run -v fix
+
+      - name: Execute deployment checks
+        if: matrix.execute-deployment == 'yes'
+        run: |
+          ./dev-tools/build.sh
+          PHP_CS_FIXER_TEST_ALLOW_SKIPPING_SMOKE_TESTS=0 vendor/bin/phpunit tests/Smoke/
+
+      - name: Execute deployment post-hook
+        if: matrix.execute-deployment == 'yes' && startsWith(github.ref, 'refs/tags/v')
+        env:
+          TRAVIS_TOKEN: ${{ secrets.TRAVIS_TOKEN }}
```

Comment:
TRAVIS_TOKEN?

Reply:
yes. a token to kick a Travis CI job for website generation.
(btw, not configured yet, gonna play around it when smoke tests are already in place)